playbooks/roles/edxapp/tasks/deploy.yml

```
@@ -289,13 +289,16 @@
 # --no-save is passed as a flag to npm install to avoid saving these dependencies to package.json. Otherwise,
 # running npm install without this flag causes modifications to the package.json and package-lock.json
 # files. In turn, these modified files cause issues with working with the edxapp repository.
+
 - name: Install private node dependencies
   shell: "easy_install --version && npm install --no-save {{ item.name }}"
-  with_items: "{{ EDXAPP_PRIVATE_NPM_REQUIREMENTS }}"
+  with_items:
+    - "{{ EDXAPP_PRIVATE_NPM_REQUIREMENTS }}"
+    - "{{ EDXAPP_PRIVATE_NPM_REQUIREMENTS_TESTING }}"
   args:
     chdir: "{{ edxapp_code_dir }}"
   become_user: "{{ edxapp_user }}"
-  when: EDXAPP_PRIVATE_NPM_REQUIREMENTS|length > 0
+  when: (EDXAPP_PRIVATE_NPM_REQUIREMENTS | length > 0) or (EDXAPP_PRIVATE_NPM_REQUIREMENTS_TESTING | length > 0)
```

I believe we can simplify this to:
```suggestion
  when: EDXAPP_PRIVATE_NPM_REQUIREMENTS or EDXAPP_PRIVATE_NPM_REQUIREMENTS_TESTING
```